test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,46 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const fixedPort = 8080;
```

Comment:
@Trott I understood why this should move to sequential. I'm not defending this, I understand why this is the case and I agree with your review. I just wanted to explain why I thought to use `8080` there. 

Frankly, I'm not really sure what happens on every platform in a server's close callback. I just thought you guys might know and determine if this is an acceptable solution. I'm satisfied with the outcome and also I've learned some things along the way.

So, thanks for that :)